gubernator/README.md

```diff
@@ -31,20 +31,7 @@ only updates after the webhook is added will be shown on the dashboard.
 
 - Get the "Gubernator Github Webhook Secret" (ask test-infra for help) and write
   it to `github/webhook_secret`.
-- Set up `secrets.json` to support Github [OAuth logins](https://github.com/settings/applications).
-  The skeleton might look like:
-
-```json
-    {
-        "k8s-gubernator.appspot.com": {
-            "session": "(128+  bits of entropy for signing secure cookies)",
-            "github_client": {
-                "id": "(client_id for the oauth application)",
-                "secret": "(client_secret for the oauth application)"
-            }
-        }
-    }
-```
+- Visit /config to support Github [OAuth logins](https://github.com/settings/applications).
```

Comment:
Sample `POST` payload would still be nice

Reply:
It requires admin authentication, so it's not very easy to script this. You'd have to open the page in your browser and extract the cookie, at which point you've saved no time over POSTing.

Reply:
Ok, then let's link to or document that flow?